### PR TITLE
New package: kblocks-20.04.0

### PR DESCRIPTION
**Pull request by toluschr**
#20892

### CODE REVIEW

File: srcpkgs/kblocks/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'kblocks'
+pkgname=kblocks
+version=20.04.0
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=off"
```
Comment (Chocimier) [inline srcpkgs/kblocks/template:6]: please try to enable test like in `ark`

**Comment by toluschr** [comment]
I enabled tests. Shouldn't they also be enabled in kmines and kapman?

**Comment by Chocimier** [comment]
Yes, please enable there on next update.

File: srcpkgs/kblocks/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+pkgname=kblocks
+version=20.04.0
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules qt5-qmake qt5-host-tools kdoctools kconfig
+ kcoreaddons gettext"
+makedepends="kconfig-devel kconfigwidgets-devel kcoreaddons-devel kcrash-devel
+ kdbusaddons-devel kdoctools-devel ki18n-devel ktextwidgets-devel kxmlgui-devel
+ libkdegames-devel kio-devel"
+short_desc="KMines is the classic Minesweeper game"
```
Comment (Chocimier) [inline srcpkgs/kblocks/template:11]: Please fix. Also no need to put package name into description.

**Comment by toluschr** [comment]
I totally forgot about the description. It's fixed now

**Comment by Chocimier** [comment]
Really no need to repeat nae in description, "Classic falling blocks game" is enough.

**Pull request merged**